avalon/tvpaint/pipeline.py

```diff
@@ -28,8 +36,145 @@ def uninstall():
 
     """
 
-    pyblish.deregister_host("tvpaint")
+    pyblish.api.deregister_host("tvpaint")
+
+
+def containerise(name, namespace, nodes, context, loader):
```

Comment:
You are claiming schema `schema": "avalon-core:container-2.0` here but not adhering to it. Id is there so we can differentiate between what is container and what is publishable instance. ObjectName in TV paint case doesn't make sense and should be replaced by UUID as you mentioned
```
  "required": [
        "schema",
        "id",
        "objectName",
        "name",
        "namespace",
        "loader",
        "representation"
    ],
```

Reply:
I just copied containerise from another host, removed few lines and then didn't get back to it as was focusing on creators. Added `"id"` and `"objectName"` is layer id (which is only reference to loaded stuff).

Reply:
I'm not sure if layer id is enough but I think 2 loaded images can't have same layer so it is actually good identifier. Will be possible that one loaded "item" will have more layers (audio 2 channels)? in that case it should be `"layer_ids"` with multiple layers.